### ISSUE TITLE
custom_pages exporting state default and no url even when a custom page is set

### ISSUE THREAD

**Issue by jurcyk** [cloudflare/cf-terraforming#222]
The resulting output from custom_pages does not reflect what is currently configured
This is a cloudflare enterprise account with 8 zones

The zone I am exporting has custom pages using this command
cf-terraforming --account ${CLOUDFLARE_ACCOUNT_ID} --zone ${ZONE_NAME} --email ${CLOUDFLARE_EMAIL} --key ${CLOUDFLARE_API_KEY} custom_pages > ${FILE}

1. Enter urls for custom pages
2. run cf-terraforming to export custom pages
3. Output has state default and no url

Example Output
```
resource "cloudflare_custom_pages" "custom_pages_basic_challenge" {
  zone_id = "ffffffffffffffffffffffffffffffff"
  type    = "basic_challenge"
  state   = "default"
}
```

Example error when running terraform plan
```
Error: Missing required argument

  on custom_pages.tf line 2, in resource "cloudflare_custom_pages" "custom_pages_basic_challenge":
   2: resource "cloudflare_custom_pages" "custom_pages_basic_challenge" {

The argument "url" is required, but no definition was found.
```

**Comment by tolidano** [comment]
I have a similar (but opposite) problem. I do NOT have any custom pages set on the Custom Pages for any of my zones but I get a file full of these (repeated for each zone):
```ccl
resource "cloudflare_custom_pages" "custom_pages_basic_challenge" {
   zone_id = "SOME_ZONE"
   type    = "basic_challenge"
   state   = "default"
 }
 
 resource "cloudflare_custom_pages" "custom_pages_waf_challenge" {
   zone_id = "SOME_ZONE"
   type    = "waf_challenge"
   state   = "default"
 }
 
 resource "cloudflare_custom_pages" "custom_pages_waf_block" {
   zone_id = "SOME_ZONE"
   type    = "waf_block"
   state   = "default"
 }
```
The URL is never there (because one isn't specified and the tool checks for that). But then what exactly is this defining?

Also, this isn't technically valid terraform - the resource names (i.e. `custom_pages_waf_block`) is repeated over and over again and the output would never plan because of this variable reuse (should it have zone in the name to disambiguate and some generic/account for the global pages maybe?)

This also ends up confusing tools like, i.e. pyhcl, as upon import, the map populates by the tuple of object type (`resource`), resource type (i.e. `cloudflare_custom_pages`), and resource name (i.e. `custom_pages_waf_block`) which means I only parse out the last zone's entries.

**Comment by jacobbednarz** [comment]
this will be addressed with some upcoming changes in #231
